msnbase2.Rnw

```diff
@@ -177,13 +177,16 @@ experimentData: use 'experimentData(object)'
 [35] "spectrum"
 @
 
-In the following sections, we describe \texttt{MSnbase}'s ability to
-efficiently handle very large mass spectrometry data and experiments
-and how it can be used for data processing and visualisation. We will
-also illustrate how it makes use of the forward-pipe operator
-(\texttt{\%>\%}) defined in the \texttt{magrittr} package. This
-operator has proved useful to develop non-trivial analyses by
-combining individual functions into easily readable pipelines.
+In the following sections, we describe \sout{\texttt{MSnbase}'s ability to
+  efficiently handle very large mass spectrometry data and experiments and how
+  it} \code{red}{how \texttt{MSnbase}} can be used for data processing and
+visualisation. We will also illustrate how it makes use of the forward-pipe
+operator (\texttt{\%>\%}) defined in the \texttt{magrittr} package. This
```

Comment:
I would remove the sentence "This operator has proved ... readable pipelines." because it is irrelevant for MSnbase and our message.

Reply:
This was part of the original text but I agree. Maybe we can then remove it when it comes to reduce/shorten the text to fit on the allowed number of pages.

Reply:
I actually think it is important enough to highlight it. I think making R code easier to read is an important feature. Given the wide usage of the tidyverse and the pipe operator, presenting some non-trivial chain of piped operations is noteworthy.